### ISSUE TITLE
Config settings not changing cloud height

### ISSUE THREAD

**Issue by 533kr** [DaFuqs/HeadInTheClouds#8]
I'm trying to raise the level where the rain/etc. stops, but even with the following in the config file:

`{
	"WorldCloudHeights": {
		"minecraft:overworld": 300.0
	}
}`

The weather still clears up at around 180-190. It seems to have no effect regardless of what I put in the config. Is there something I'm missing? I have a huge selection of mods but the only mods effecting weather/clouds is Primal Winter.

**Comment by DaFuqs** [comment]
- Clients (cloud rendering, rain gradient, ...): The value is directly pulled from the client world itself that tells HeadInTheClouds where the clouds are. The config does not have any effect. So since the overworld renders its clouds at y=182, HeadInTheClouds always uses that value. That means it automatically works in all modded dimensions.
- Servers  (farmland hydration, ...): The value in the config is used, since the server (including the local builtin server on clients) has no idea about the cloud height (it does not have to render clouds, so it does not know where they would be). Therefore HeadInTheClouds needs to be told in the config explicitly

TL;DR:
- The height where rain is cleared gets fetched from the worlds internals (that is provided by vanilla & mods). The config only takes effect on servers. (internally called "[dimension effects](https://minecraft.wiki/w/Effect_(dimension)")
